src/schema/posts.ts

```diff
@@ -2344,19 +2345,21 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       ctx: AuthContext,
       info,
     ): Promise<GQLPost> => {
-      await Promise.all([
-        ctx.con.getRepository(Post).findOneByOrFail({ id }),
+      const [post] = await Promise.all([
+        ctx.con.getRepository(Post).findOneOrFail({ where: { id } }),
```

Comment:
why even this change, is it not the same thing `findOneByOrFail` or `findOneOrFail`?

Reply:
It was on the try above for select, but figured it won't work due to different entities.
https://github.com/dailydotdev/daily-api/pull/2627#discussion_r1935177097